Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -46,12 +47,17 @@ final class SettingsTableViewController: UITableViewController {
 
     var dataManager: DeviceDataManager!
 
+    private lazy var isTestingPumpManager = dataManager.pumpManager is TestingPumpManager
+    private lazy var isTestingCGMManager = dataManager.cgmManager is TestingCGMManager
+
     fileprivate enum Section: Int, CaseCountable {
         case loop = 0
         case pump
         case cgm
         case configuration
         case services
+        case testingPumpDataDeletion
```

Comment:
If you only have a testing CGM, the section for deleting pump data is shown instead of the one for deleting cgm data.

Reply:
Oops, I had this fix committed locally and forgot to push it. Thanks!